packages/client/src/atoms/ListItem.ts

```diff
@@ -2,6 +2,8 @@ import { ListItem } from "@amsterdam/asc-ui";
 import styled from "styled-components";
 
 export default styled(ListItem)`
+  font-size: 16px;
```

Comment:
Can you not make this optional via a prop? Because now all ListItems will be this size?